android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java

```diff
@@ -623,6 +640,21 @@ public boolean skipBlankSteps() {
         }
     }
 
+    @Override
+    public void makeFieldHidden(JSONObject fieldObject) {
+        Utils.makeFieldHidden(fieldObject);
+    }
+
+    @Override
+    public void makeFieldDisabled(JSONObject fieldObject) {
+        Utils.makeFieldDisabled(fieldObject);
+    }
```

Comment:
This implementations are not needed since whatever they are returning is already a `public static` method in the `Utils`